domain/src/main/kotlin/no/nav/su/se/bakover/domain/brev/LagBrevRequest.kt

```diff
@@ -375,12 +384,13 @@ interface LagBrevRequest {
     }
 }
 
-fun LagBrevRequest.lagPersonalia() = this.person.let {
+fun LagBrevRequest.lagPersonalia(saksnummer: Saksnummer) = this.person.let {
```

Comment:
Hvis du har lagt in `val saksnummer` i interface:t `LagBrevRequest`, så skjønner jag ikke helt hvorfor du har lust å sende in den som parameter og här? Kunne du ikke bara tatt bort `saksnummer` sån som den er?

Reply:
enig. endret nå